ui/lib/sync/addon/components/secrets/page/destinations/destination/secrets.ts

```diff
@@ -39,9 +39,13 @@ export default class SyncSecretsDestinationsPageComponent extends Component<Args
   async update(association: SyncAssociationModel, operation: string) {
     try {
       await association.save({ adapterOptions: { action: operation } });
-      // this message can be expanded after testing -- deliberately generic for now
-      this.flashMessages.success(
-        'Sync operation successfully initiated. Status will be updated on secret when complete.'
+      const action: string = operation === 'set' ? 'Sync' : 'Unsync';
+      this.flashMessages.success(`${action} operation initiated.`);
+      // refresh route to update list of secrets, clearing dataset does not reload UI
+      this.router.transitionTo(
```

Comment:
Kind of wonky - open to doing this different. Could bubble up to the router fire with a model refresh action, or maybe refactor so the list updates when the args change?